src/sensor.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+)
+
+func main() {
+	ch := make(chan string)
+
+	go getReadings(ch)
+	fmt.Println(<-ch)
+}
+
+func getReadings(ch chan<- string) {
+	// generate readings
+	start := time.Now()
+  rand.Seed(start.UTC().UnixNano())
```

Comment:
We have to align our tab indentation :s

Reply:
I propose we use 2 tab spacing since it makes it harder to reach a width threshold

Reply:
That sounds nice. Can you change it for us? I'll set it in my vscode.

Reply:
Alright!